src/Game.cxx

```diff
@@ -370,3 +358,46 @@ template <typename MQType, typename Visitor> void Game::LoopMain(GameContext &co
     // @todo: Call shutdown() here in a safe way
   }
 }
+
+void gameLoop(void* arg_)
```

Comment:
It's good that you were able to get it to work this way. However I don't think this is the right solution. If Emscripten truly doesn't support multithreading then we can try using coroutines instead. For now, this is a good starting point

Reply:
i was sure you wouldn't be happy with this solution. (Because i'm not too) According to the documentation:

https://emscripten.org/docs/porting/emscripten-runtime-environment.html#browser-main-loop

it seems to be the only way to do this.
I think we could ifdef this?